ontology/yaml/resources/SAFETY/entity_types/ABSTRACT.yaml

```diff
@@ -86,11 +88,27 @@ WLS:
   is_abstract: true
   uses:
   - water_leak_cable_distance_sensor
-  
+
 LM:
   guid: "25887ceb-f2d8-4b25-be80-36e8b37f5ae9"
   description: "Tank level monitoring."
   is_abstract: true
   uses:
-  - level_status 
+  - level_status
 
+MDC:
+  guid: "ec1aab74-c961-40fd-87b7-ce219c9c6d3a"
+  description: "Tag for those with motorized dampers"
```

Comment:
Please revise the description to be more generic; something like: "Motorized binary damper control."

Reply:
Noted ma'am, will do.